From b3fdf5b83dc79575405f524df889d92908fecc72 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Marcus=20M=C3=BCller?= <Marcus.Mueller@ettus.com>
Date: Thu, 21 May 2015 23:46:58 -0700
Subject: [PATCH 031/131] Fix config error with CMake version 2.8.7

Replace find_package(SoapySDR CONFIG) with find_package(SoapySDR NO_MODULE),
which has an identical effect but is supported by CMake 2.8.7 on Ubuntu 12.10
---
 CMakeLists.txt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index f715414..f0b1283 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -165,7 +165,7 @@ endif(ENABLE_NONFREE)
 find_package(LibHackRF)
 find_package(LibAIRSPY)
 find_package(LibbladeRF)
-find_package(SoapySDR CONFIG)
+find_package(SoapySDR NO_MODULE)
 find_package(Doxygen)
 
 if(NOT GNURADIO_RUNTIME_FOUND)
-- 
2.20.1

